tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/PitclipseUiRunnerTest.java

```diff
@@ -34,7 +34,6 @@ public void emptyClassAndEmptyTest() throws CoreException {
         runTest(FOO_TEST_CLASS, FOO_BAR_PACKAGE, TEST_PROJECT);
         consoleContains(0, 0, 100, 0, 0);
         mutationsAre(Collections.emptyList());
-        coverageReportGenerated(0, 100, 100);
```

Comment:
From what I understand you removed these calls because with the new PIT in this case no HTML is generated right? But what happens if you leave these calls (that is, the developer inadvertently calls that or a test that was succeeding now fails because of a breaking change in the code)? Exception? Immediate failure? Timeout?

Reply:
They run into a timeout, if they were left in place.

>
>[...] or a test that was succeeding now fails because of a breaking change in the code)? Exception? Immediate failure? Timeout?
I don't understand what you mean with that.